catalog/video/video-security/pom.xml

```diff
@@ -90,17 +95,17 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.81</minimum>
+                                            <minimum>0.14</minimum>
```

Comment:
❓ These numbers look like they dropped a lot considering the changes you made. Just want to bring some attention to it 😄